superset-frontend/src/explore/components/PropertiesModal/index.tsx

```diff
@@ -54,7 +49,7 @@ export default function PropertiesModal({
   const [cacheTimeout, setCacheTimeout] = useState(
     slice.cache_timeout != null ? slice.cache_timeout : '',
   );
-  const [owners, setOwners] = useState<OptionsType<OwnerOption> | null>(null);
+  const [owners, setOwners] = useState<SelectValue | null>(null);
```

Comment:
Should we call this `selectedOwners`? At first, I thought it was conflicting with `loadOptions`.

Reply:
> Should we call this `selectedOwners`? At first, I thought it was conflicting with `loadOptions`.

That's a really good idea 👍  this would make it more explicit and less likely for other developers misunderstanding this piece of state.